app/src/main/java/com/keylesspalace/tusky/MainActivity.kt

```diff
@@ -0,0 +1,606 @@
+                        val intent = Intent(context, SavedTootActivity::class.java)
+                        startActivityWithSlideInAnimation(intent)
+                    }
+                    DRAWER_ITEM_SCHEDULED_TOOT -> {
+                        startActivityWithSlideInAnimation(ScheduledTootActivity.newIntent(context))
+                    }
+                    DRAWER_ITEM_LISTS -> {
+                        startActivityWithSlideInAnimation(ListsActivity.newIntent(context))
+                    }
+                }
+                true
+            }
+            setSavedInstance(savedInstanceState)
+        }
+
+        if (BuildConfig.DEBUG) {
+            mainDrawer.addItems(
+                    SecondaryDrawerItem().apply {
+                        identifier = 1337
+                        nameText = "debug"
+                        isEnabled = false
+                        textColor = ColorStateList.valueOf(Color.GREEN)
+                        isSelectable = false
+                    }
+            )
+        }
+        EmojiCompat.get().registerInitCallback(emojiInitCallback)
+    }
+
+    override fun onConfigurationChanged(newConfig: Configuration) {
+        super.onConfigurationChanged(newConfig)
+        drawerToggle.onConfigurationChanged(newConfig)
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        if (drawerToggle.onOptionsItemSelected(item)) {
+            return true
+        }
+        return super.onOptionsItemSelected(item)
+    }
+
+    override fun onSaveInstanceState(outState: Bundle) {
+        super.onSaveInstanceState(mainDrawer.saveInstanceState(outState))
+    }
+
+    private fun setupTabs(selectNotificationTab: Boolean) {
+        val tabs = accountManager.activeAccount!!.tabPreferences
+        adapter = MainPagerAdapter(tabs, this)
+        viewPager.adapter = adapter
+        TabLayoutMediator(tabLayout, viewPager, TabConfigurationStrategy { _: TabLayout.Tab?, _: Int -> }).attach()
+        tabLayout.removeAllTabs()
+        for (i in tabs.indices) {
+            val tab = tabLayout.newTab()
+                    .setIcon(tabs[i].icon)
+            if (tabs[i].id == LIST) {
+                tab.contentDescription = tabs[i].arguments[1]
+            } else {
+                tab.setContentDescription(tabs[i].text)
+            }
+            tabLayout.addTab(tab)
+            if (tabs[i].id == NOTIFICATIONS) {
+                notificationTabPosition = i
+                if (selectNotificationTab) {
+                    tab.select()
+                }
+            }
+        }
+    }
+
+    private fun handleProfileClick(profile: IProfile, current: Boolean): Boolean {
+        val activeAccount = accountManager.activeAccount
+
+        //open profile when active image was clicked
+        if (current && activeAccount != null) {
+            val intent = AccountActivity.getIntent(this, activeAccount.accountId)
+            startActivityWithSlideInAnimation(intent)
```

Comment:
`new Handler().postDelayed(() -> drawer.closeDrawer(), 100);` is gone, is this intended?

Reply:
yes